site/gatsby-site/server/rules.ts

```diff
@@ -22,4 +27,28 @@ export const isRole = (role: string) => rule()(
     },
 )
 
+export const isSelf = () => rule()(
```

Comment:
My idea here is that only `admin` users can query data about a user other than themselves.

We might need to tweak this in the future so `first_name` and `last_name` are public.